app.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/app.controller");
+
+app.use(express.json());
```

Comment:
app.use(express.json()) -> let's remove this until we need it for a post/patch request